scripts/kani-run-on-repos.sh

```diff
@@ -0,0 +1,136 @@
+#!/bin/bash
```

Comment:
I'm wondering if we should create a new `scripts/exps/` folder for these experimental scripts. What do you think?

Reply:
I don't have strong preference in this area. It depends on whether to consider this as a metric or a test case.

Reply:
Also, do we have other scripts than this one to put in such a directory?

Reply:
I think this is going to work mostly as a metric. My concern here is that `scripts` is becoming too cluttered, and I'd propose moving "format" scripts to `ci` and regression scripts to a new folder `tests`. There are no other scripts that would go into this directory, at least for now.

Reply:
I think that is reasonable. Fixed w/ a067c8d6e071652fdad5eefa6efd2e3cf3254734